config.toml

```diff
@@ -37,8 +37,8 @@ themesDir = "node_modules/"
   tag = "tags"
 
 [Author]
-  name = "OpenHW Group"
-  website = "https://www.openhwgroup.org"
+  name = "OpenHW Foundation"
+  website = "https://openhwfoundation.org"
   email = "info@openhwgroup.org"
```

Comment:
I'll need to confirm whether the new domain is set up for emails.

Reply:
Did you reach out to Shanda or Sharon for this?